pkgs/by-name/ya/yalc/package.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchYarnDeps,
+  nodejs,
+  yarnBuildHook,
+  yarnConfigHook,
+  yarnInstallHook,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "yalc";
+  version = "1.0.0-pre.54-unstable-2023-07-04";
+
+  src = fetchFromGitHub {
+    owner = "wclr";
+    repo = "yalc";
+    # Upstream has no tagged versions
+    rev = "3b834e488837e87df47414fd9917c10f07f0df08";
+    hash = "sha256-v8OhLVuRhnyN2PrslgVVS0r56wGhYYmjoz3ZUZ95xBc=";
+  };
+
+  yarnOfflineCache = fetchYarnDeps {
+    yarnLock = "${src}/yarn.lock";
+    hash = "sha256-+w3azJEnRx4v3nJ3rhpLWt6CjOFhMMmr1UL5hg2ZR48=";
+  };
+
+  nativeBuildInputs = [
+    yarnConfigHook
+    yarnBuildHook
+    yarnInstallHook
+    nodejs
+  ];
+
+  meta = {
+    description = "Work with yarn/npm packages locally like a boss.";
```

Nitpick: [pkgs/README.md](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes) says that ending punctuation should not be present.
```suggestion
    description = "Work with yarn/npm packages locally like a boss";
```

---

That one always catches me, thanks!